api-client/src/main/java/com/xing/api/internal/json/CsvCollectionJsonAdapter.java

```diff
@@ -33,11 +33,14 @@
 
 /** Adapter that parses comma separated value strings into a collection. */
 public abstract class CsvCollectionJsonAdapter<C extends Collection<String>> extends JsonAdapter<C> {
-    private static final String COMMA_DELIMITER = ", ";
+    private static final String COMMA_DELIMITER = ",";
     private static final Pattern COMMA_SEPARATOR = Pattern.compile(COMMA_DELIMITER);
 
+    private static final String COMMA_SPACE_DELIMITER = ", ";
+    private static final Pattern COMMA_SPACE_SEPARATOR = Pattern.compile(COMMA_SPACE_DELIMITER);
+
     /** Comma separated values adapter factory. */
-    public static final JsonAdapter.Factory FACTORY = new Factory() {
+    public static final Factory FACTORY = new Factory() {
```

Comment:
Please revert to  `public static final JsonAdapter.Factory FACTORY = new Factory() {`